react-native/training/src/app/screens/ListBooks/components/detail/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Image, View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import CustomText from '../../../../components/CustomText';
+import defaultImage from '../../../../../assets/default-image-user.png';
```

Comment:
You use this twice. Don't you think that is better to put this in a `constant` file?